docs/reference/logging/splunk.md

```diff
@@ -32,16 +32,22 @@ You can set the logging driver for a specific container by using the
+|-----------------------------|----------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `splunk-token`              | required | Splunk HTTP Event Collector token.                                                                                                                                                                                 |
+| `splunk-url`                | required | Path to your Splunk Enterprise or Splunk Cloud instance (including port and schema used by HTTP Event Collector) `https://your_splunk_instance:8088`.                                                              |
+| `splunk-source`             | optional | Event source.                                                                                                                                                                                                      |
+| `splunk-sourcetype`         | optional | Event source type.                                                                                                                                                                                                 |
+| `splunk-index`              | optional | Event index.                                                                                                                                                                                                       |
+| `splunk-capath`             | optional | Path to root certificate.                                                                                                                                                                                          |
+| `splunk-caname`             | optional | Name to use for validating server certificate; by default the hostname of the `splunk-url` will be used.                                                                                                           |
+| `splunk-insecureskipverify` | optional | Ignore server certificate validation.                                                                                                                                                                              |
+| `tag`                       | optional | Specify tag for message, which interpret some markup. Default value is `{{.ID}}` (12 characters of the container ID). Refer to the [log tag option documentation](log_tags.md) for customizing the log tag format. |
+| `labels`                    | optional | Comma-separated list of keys of labels, which should be included in message, if these labels are specified for container.                                                                                          |
```

Comment:
I see some other drivers mention them as well. I'm good with leaving them, and we can do a cleanup / restructure in a later stage.